gradle/license-header.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2018-present The Code Analysis Annotations Authors.
```

Comment:
- Change "The Code Analysis Annotations Authors" to "The jspecify Authors."

- I don't know much about copyright statements, but our internal guidelines recommend a specific year rather than "present." I see a similar recommendation to avoid "present" in [this random Stack Exchange answer](https://opensource.stackexchange.com/a/4890/17871).